flink-connectors/flink-connector-base/src/test/java/org/apache/flink/connector/base/source/reader/CoordinatedSourceRescaleITCase.java

```diff
@@ -122,8 +121,9 @@ private StreamExecutionEnvironment createEnv(
         env.setParallelism(p);
         env.enableCheckpointing(100);
         env.getCheckpointConfig()
-                .setExternalizedCheckpointCleanup(
-                        CheckpointConfig.ExternalizedCheckpointCleanup.RETAIN_ON_CANCELLATION);
+                .setExternalizedCheckpointCleanupRetention(
```

Comment:
Actually I'd prefer `setExternalizedCheckpointRetention` @masteryhx WDYT?

Reply:
+1, This is consistent with real config name.
BTW, I'd also suggest to rename the new class name `ExternalizedCheckpointCleanup` or at least add some descriptions e.g. 'Also called ExternalizedCheckpointRetention' in java doc and doc to link them for the convinency of users.
WDYT?

Reply:
I think it is more reasonable to change the class name to `ExternalizedCheckpointRetention`

Reply:
@masteryhx Class name has been modified